ballerina/Module.md

```diff
@@ -24,31 +24,31 @@ Follow one of the following ways to add the JAR in the file:
     ```
 
 ### Client
-To access a database, you must first create a
+To access a database, you must first create an
 [`mssql:Client`](https://docs.central.ballerina.io/ballerinax/mssql/latest/clients/Client) object.
-The examples for creating a MSSQL client can be found below.
+The examples for creating an MSSQL client can be found below.
 
 #### Creating a Client
-This example shows the different ways of creating the `mssql:Client`.
+These examples show the different methods of creating an `mssql:Client`.
 
-The client can be created with an empty constructor, and thereby, the client will be initialized with the default properties.
+The client can be created with an empty constructor, and thereby, will be initialized with the default properties.
 
 ```ballerina
 mssql:Client|sql:Error dbClient = new();
 ```
 
-The `dbClient` receives the host, username, and password. Since the properties are passed in the same order as they are defined
-in the `mssql:Client`, you can pass them without named params.
+The `mssql:Client` receives the host, username, and password. Since the properties are passed in the same order as they are defined
+in the `mssql:Client`, you can pass them without named parameters.
 
 ```ballerina
 mssql:Client|sql:Error dbClient = new(
    "localhost", "rootUser", "rootPass", "information_schema", 1443
 );
 ```
 
-The `dbClient` uses the named params to pass the attributes since it is skipping some params in the constructor.
+The sample below shows as `mssql:Clinet` using named parameters to pass the attributes since some parameters are skipped in the constructor.
```

Comment:
```suggestion
The sample below shows as `mssql:Client` using named parameters to pass the attributes since some parameters are skipped in the constructor.
```